live-examples/js-examples/date/date-toprimitive.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">
+  const date = new Date('20 December 2019 14:48'); //initialize a new Date object
+
+  console.log(date[Symbol.toPrimitive]('string')); //log primitive of date with hint as string
+  //expected output: "Fri Dec 20 2019 20:18:00 GMT+0530 (India Standard Time)"
```

Comment:
There should be a space between `//` and the comment:

`// expected output: "Fri Dec 20 2019 20:18:00 GMT+0530 (India Standard Time)"`